controller/k8s/api.go

```diff
@@ -47,6 +49,7 @@ const (
 	SP
 	SS
 	Svc
+	Job
```

Comment:
mind maintaining the sort order? also check the other places where `Job` appears in a list in the code.